app/Models/LegiScan/Bill/History.php

```diff
@@ -128,9 +128,11 @@ public function getNebraskaHearingDateAttribute()
 
     public function getNebraskaHearingDateHumanizedAttribute()
     {
-        return $this->nebraska_hearing_date
-             ? $this->nebraska_hearing_date->diffForHumans(Carbon::now('utc'), CarbonInterface::DIFF_RELATIVE_TO_NOW)
-             : null;
+        return Str::replace(
+            '1 second ago',
+            'Today',
+            $this->nebraska_hearing_date?->diffForHumans(Carbon::now()->startOfDay(), CarbonInterface::DIFF_RELATIVE_TO_NOW)
```

Comment:
Still missing the point. You're altering the timestamp beyond recognition here and there's no reason to believe that what you're outputting at any given time of day is the correct relative date the way you're doing it.

<img width="1408" alt="image" src="https://user-images.githubusercontent.com/963674/214717891-2f5abacd-ad2f-4c51-9457-c882c3588dc3.png">

It's likely (although this is a complete guess) that this is because of Timezone info embedded into the datetime field. I have no idea w/out seeing an example. However, in this case, this is still not the way to do it, because again, your output the way you're doing it sis - and this is important and worth repeating - **relative to the time of day the user visits the page.** In other words, if they visit at 10am and then later at 7pm, the day may bounce forward by a day even though **the database value hasn't changed**.

What you want to do is to strip TZ data _out_ of the datetime value and then display the diff relative to now. For example

```
Carbon::parse($date->toDateString())->diffForHumans
```

`$date->toDateString()` means that no matter what TZ the datetime is in relative to server time, the diff will simply be calculated off of the raw, TZ-agnostic, calendar date. Either that or you need to convert the datetime to your TZ, but I severely doubt that this is actually what you're trying to do here. But even then the syntax is simpler than what you're doing.

I'll repeat again: What you're doing here is **not normal** and it's not syntax you should be repeating. This complexity should be a yellow flag in your brain that you're doing something non-standard and weird.

Reply:
They are both UTC time. The reason for setting both to `startOfDay()` is because the distance will be an entire day.

I'm pretty sure it's working now. 

Think of it like  measuring a wave, we want to measure from peak to peak.

If I just do a diff from `now()` it will be for the middle of the day and that doesn't work.

Reply:
The point of my example is not to do this from `now()` The point is how much dither there is the way you're doing it. If you need start of day, just call it on the date without all this interface nonsense.

<img width="819" alt="image" src="https://user-images.githubusercontent.com/963674/214882107-ac53e2ac-b1a1-4a42-928a-e6aba4b8f06d.png">